L/LLVM/common.jl

```diff
@@ -273,6 +307,11 @@ CMAKE_FLAGS+=(-DCMAKE_CXX_COMPILER_TARGET=${CMAKE_TARGET})
 CMAKE_FLAGS+=(-DCMAKE_ASM_COMPILER_TARGET=${CMAKE_TARGET})
 
 cmake -GNinja ${LLVM_SRCDIR} ${CMAKE_FLAGS[@]} -DCMAKE_CXX_FLAGS="${CMAKE_CPP_FLAGS} ${CMAKE_CXX_FLAGS}" -DCMAKE_C_FLAGS="${CMAKE_CPP_FLAGS} ${CMAKE_CXX_FLAGS}"
+if [[ "${target}" == *-mingw* ]]; then
+    # Build system for Windows seems to be plagued by
+    # race conditions, reduce parallelism to limit them
+    (( nproc=1+nproc/3 ))
```

Comment:
This didn't help. I have really no idea if what to do else with the Windows build.